modes/play/objects/badGuy.js

```diff
@@ -0,0 +1,35 @@
+class BadGuy extends Mob {
+  constructor(game, mode, x, y) {
+    super(game, mode, x, y);
+    this.color = "#ee5";
+    this.brain = new BadGuyBrain(this);
+  }
+}
+
+
+class BadGuyBrain {
+  constructor(meat) {
```

Comment:
The "body" of the mob may not actually be made out of meat. The class is designed to be flexible enough that you can rename that parameter to whatever you want: `chassis`, `vehicle`, `body`, whatever.